src/libs/actions/IOU.ts

```diff
@@ -6423,9 +6423,8 @@ function setSplitShares(transaction: OnyxEntry<OnyxTypes.Transaction>, amount: n
         }
 
         const isPayer = accountID === userAccountID;
-
-        // This function expects the length of participants without current user
-        const splitAmount = IOUUtils.calculateAmount(accountIDs.length - 1, amount, currency, isPayer);
+        const participantsLength = newAccountIDs.includes(userAccountID) ? newAccountIDs.length - 1 : newAccountIDs.length;
```

Comment:
Context, `newAccountIDs` may or may not contain the current user accountID depending on where this is called from. If we're splitting from within an existing group, it will have the user's accountID. If from Global Create, it won't. We need to send the participants length - 1 to `calculateAmount`